mypy/nodes.py

```diff
@@ -390,6 +405,10 @@ def set_line(self, target: Union[Token, Node, int]) -> Node:
             self.initialization_statement.set_line(self.line)
             self.initialization_statement.lvalues[0].set_line(self.line)
 
+    def set_column(self, target: Union[Token, Node, int]) -> Node:
+        super().set_column(target)
+        return self
```

Comment:
just matching behaviour of set_line - not sure of philosophy around chaining generally

there are some places in the code where already we do
`some_thing = SomeNode(blah).set_line(1)`
so with this they become
`some_thing = SomeNode(blah).set_line(1).set_column(2)`